charts/generic-govuk-app/templates/rails-secret-key-base-external-secret.yml

```diff
@@ -0,0 +1,24 @@
+{{- if (and .Values.rails.enabled .Values.rails.createKeyBaseSecret) }}
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: {{ .Values.repoName }}-rails-secret-key-base
```

Comment:
I guess we need a default for repoName to satisfy kubeconform? I'm not sure where kubeconform is currently reading the Helm values from though. Presumably the default values.yaml for the generic-govuk-app chart?

Reply:
Yeah, so ended up adding that values file. I was worried about it overriding [the default where repoName = name](https://github.com/alphagov/govuk-helm-charts/blob/dfd5a9a53ad1bc938dd07a4e5bff49776ef44764/charts/app-config/templates/govuk-application.yaml#L2) - but then realised that is calculated separately.